locales/en-US/app.ftl

```diff
@@ -36,7 +36,7 @@ headerLinkBlog = Blog
 landingIntroOne = Test new features.
 landingIntroTwo = Give your feedback.
 landingIntroThree = Help build Firefox.
-landingLegalNotice = By proceeding, you agree to the <a>Terms of Use</a> and <a>Privacy Notice</a> of Test Pilot.
+landingLegalNotice = By proceeding, you agree to the <terms-link>Terms of Use</terms-link> and <privacy-link>Privacy Notice</privacy-link> of Test Pilot.
```

Comment:
This one still needs a new ID.

Reply:
🤦‍♂️ I think I accidentally didn't commit this. Sorry, of course it needs a new id.